src/server/node_recov_db.c

```diff
@@ -88,152 +83,187 @@
 #include "libutil.h"
 #include "pbs_db.h"
 
-#ifdef NAS /* localmod 005 */
-/* External Functions Called */
-extern int recov_attr_db(pbs_db_conn_t *conn,
-	void *parent,
-	pbs_db_attr_info_t *p_attr_info,
-	struct attribute_def *padef,
-	struct attribute *pattr,
-	int limit,
-	int unknown);
-extern int recov_attr_db_raw(pbs_db_conn_t *conn,
-	pbs_db_attr_info_t *p_attr_info,
-	pbs_list_head *phead);
-#endif /* localmod 005 */
+
+/**
+ * @brief
+ *		convert from database to node structure
+ *
+ * @param[out]	pnode - Address of the node in the server
+ * @param[in]	pdbnd - Address of the database node object
+ *
+ * @return	Error code
+ * @retval   0 - Success
+ * @retval  -1 - Failure
+ *
+ */
+static int
+db_2_node(struct pbsnode *pnode, pbs_db_node_info_t *pdbnd)
+{
+	if (pdbnd->nd_name && pdbnd->nd_name[0] != 0) {
+		pnode->nd_name = strdup(pdbnd->nd_name);
+		if (pnode->nd_name == NULL)
+			return -1;
+	}
+	else
+		pnode->nd_name = NULL;
+
+	if (pdbnd->nd_hostname && pdbnd->nd_hostname[0]!=0) {
```

Comment:
Put a space around !=0 between pdbnd->nd_hostname[0] and 0.

Reply:
done